product/specs/2022-01-views.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Views in Mathesar
+description: Spec for Views in Mathesar
+published: true
+date: 2022-01-13T19:49:54Z
+tags: 
+editor: markdown
+dateCreated: 2022-01-13T19:49:54Z
+---
+
+**Date**: 2022-01
+**Author**: Kriti Godey
+
+# Introduction
+
+Fundamentally, **Views** are saved database queries. This means that in order to work with Views in Mathesar, we need to translate every concept that can be used in [PostgreSQL queries](https://www.postgresql.org/docs/14/queries.html) to our end users in a user-friendly way.
```

Comment:
I figured we'd be able to break down views previously defined by some DB query into the concepts defined in this spec (as long as we have access to the query, which I assume we would if we had access to the view). Is this inaccurate?

Reply:
That might be possible, but would be very (_very_) difficult under all but the simplest circumstances.  Moreover, my point is that even if we could do that, we don't need to in order to be able to work with a view.  Having access to the underlying query isn't necessary, since we can treat a view as a table for any of the operations we support.